### DISCUSSION TITLE
Accessing a row in joined table in policy

### DISCUSSION THREAD

**Discussion by aquadesk** [supabase/supabase#8783] [Questions] [answered]
Hi team,

Just want to get an idea what is the limit of the policy and see if there is any advanced way supporting my use-case.

Let's say I have tables Product and reviews and they are in 1:1 relationship.

I am trying to create a policy where reviews is editable only when associated Product is in "live" state.

Would it be possible to achieve this in Policy or I should use alternative?

I could not answer it myself going through the doc https://supabase.com/docs/learn/auth-deep-dive/auth-policies

Thanks

**Comment by Albert-Gao** [comment, marked as answer]
is this what you are looking for?

https://supabase.com/docs/guides/auth/row-level-security#policies-with-joins

```sql
create policy "Team members can update team details if they belong to the team."
  on teams
  for update using (
    auth.uid() in (
      select user_id from members
      where team_id = id
    )
  );
```

**Reply by aquadesk** [reply]
Yes indeed. Thanks for the answer!
